src/components/vm/vmReplaceSpiceDialog.jsx

```diff
@@ -58,9 +70,44 @@ export const ReplaceSpiceDialog = ({ vmName, vmId, connectionName, vmRunning })
                 .finally(() => setInProgress(false));
     }
 
+    let vmSelect = null;
+
+    if (isMultiple) {
+        const onSelect = (_ev, item) => setSelected(
+            selected.includes(item)
+                ? selected.filter(id => id !== item)
+                : [...selected, item]
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1416-20240205-065324-d45075bd-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)